src/angular-business/notification/notification/notification-config.ts

```diff
@@ -0,0 +1,29 @@
+import { TemplateRef, ViewContainerRef } from '@angular/core';
+
+import { NotificationType } from '../notification-container/notification-container.component';
+
+import { JumpMark } from './notification.component';
+
+/** Possible values for horizontalPosition on NotificationConfig. */
+export type NotificationHorizontalPosition = 'center';
+
+/** Possible values for verticalPosition on NotificationConfig. */
+export type NotificationVerticalPosition = 'top' | 'bottom';
+
+export class NotificationConfig<D = any> {
```

Comment:
Why did you remove properties when comparing with MatSnackBarConfig?
Also there is no documentation on the properties?

Reply:
I opted to remove all functionalities non-specificated in the open issue / functionalities not yet provided by the "old" notification component. If you'd like a 1/1 reproduction of the SnackBar functionalities, I can still add them.

Reply:
I see.
Please re-add `politeness`, `announcementMessage` (replacing `message`) and `duration`. We should try to provide accessibility options where possible. (Also in `open` apply `message` to `announcementMessage`, if `announcementMessage` is empty as done in snack-bar.)
Also remove `jumpMarks`, `icon` and `readonly` for now as I don't want to provide this functionality for the toast.
Also please add documentation to the properties.